### PR TITLE
HPCC-12862 Need to support nothor and noroxie etc. as option on //version

### PR DESCRIPTION
**Pull request by AttilaVamos**
Add code to process exclusion(s) defined in //version tag

Update README.rst

Signed-off-by: Attila Vamos attila.vamos@gmail.com

### CODE REVIEW

**Comment by AttilaVamos** [comment]
@ghalliday please check and merge.

File: testing/regress/hpcc/util/ecl/file.py [hunk 1]
```diff
@@ -323,7 +323,7 @@ def testVesion(self):
             eclText = open(self.getEcl(), 'rb')
             for line in eclText:
                 if tag in line.lower():
-                    items = line.replace(tag, '').strip().replace('"', '')
+                    items = line.replace(tag, '').strip().replace('"', '').replace(' ', '')
```
Comment (ghalliday) [inline testing/regress/hpcc/util/ecl/file.py]: Why this change?  I assume it is to allow , nothor.

  It would cause issues if you had
//version text='hello world'

I'm also not quite sure why you strip "

Although the removal of the -Dnothor is nice it probably isn't a requirement
